template/build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
-	id 'org.springframework.boot' version "$springBootVersion"
-	id 'io.spring.dependency-management' version "$springDependencyManager"
+	id 'org.springframework.boot' version '2.2.6.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.9.RELEASE'
```

Comment:
Honestly simply because it produced errors when building the lib and this was a working quick fix...

Now that I put some time into googling the error I found that it is connected to my slightly outdated gradle version (I use 5.4.1):
https://github.com/gradle/gradle/issues/1697#issuecomment-506910915
https://docs.gradle.org/5.6/release-notes.html#central-management-of-plugin-versions-with-settings-script

So yeah  -  I will revert this change and just update my local gradle version ^^